### PR TITLE
Website2.1

### PR DESCRIPTION
**Pull request by CAnorbo**
New version of the website for the 2.0.8. release

### CODE REVIEW

File: wwwroot/about.html [hunk 1]
```diff
@@ -173,12 +233,16 @@ <h2 class="title text-center">The history of Wasabi Wallet</h2>
                                 <div class="timeline-title">2022 - <img src="img/logo_small.svg" class="timeline-small-img" alt="Wasabi Wallet"> WASABI WALLET 2.0</div>
                                 <p>After two years of development, 2.0 is launched on June 15th initiating a new era for Bitcoin privacy.</p>
                             </li>
+                            <li>
+                                <div class="timeline-title">2024 - zkSNACKs is shuting down it's coordinator.</div>
```
Comment (MarnixCroes) [inline wwwroot/about.html:237]:
```suggestion
                                <div class="timeline-title">2024 - zkSNACKs is shutting down it's coordinator.</div>
```

File: wwwroot/index.html [hunk 2]
```diff
@@ -1,257 +1,659 @@
...
+            <div class="container">
+                <div class="row align-items-center mt-5 mt-lg-0">
+                    <div class="col-lg-11 offset-lg-1">
+                        <div class="row align-items-top mb-5">
+                            <div class="col-lg-6 order-1">
+                                <h3 class="title-huge">DOWNLOAD</h3>
+                            </div>
+                            <div class="col-6 order-3 order-lg-2">
+                                <div class="ps-0 ps-lg-5">
+                                    <p class="small-text"> <a href="https://github.com/WalletWasabi#build-from-source-code" target="_blank"><i class="muted-text">or build it from source</i></a>
```
Comment (MarnixCroes) [inline wwwroot/index.html:443]: link doesn't work
```suggestion
                                    <p class="small-text"> <a href="https://github.com/WalletWasabi/WalletWasabi#build-from-source-code" target="_blank"><i class="muted-text">or build it from source</i></a>
```

File: wwwroot/index.html [hunk 3]
```diff
@@ -1,257 +1,659 @@
...
-        <section class="home p-100">
+        <section id="coinjoin" class="coinjoin-section">
             <div class="container">
-                <div class="row align-items-center">
-                    <div class="col-xl-8 offset-xl-2 text-center">
-                        <h2 class="title-small title-lined my-5">Why Wasabi Wallet?</h2>
+                <div class="row">
+                    <div class="col-lg-3 offset-lg-1">
+                        <h2 class="title">A coinjoin is...</h2>
+                        <a href="coinjoin.html" class="btn btn-secondary affix-arrowright mt-4">Read more</a>
```
Comment (MarnixCroes) [inline wwwroot/index.html:249]: this _read more_ link is dead

**Review by MarnixCroes — CHANGES_REQUESTED** [review]
first quick review

- `Privacy policy` at the menu should be removed. it links to the T&Conditions which are removed
- FAQ and Download are duplicate at home page
- what's the point of this?
image

**Comment by MarnixCroes** [comment]
`what are the fees for coinjoins` FAQ should also be updated/removed

**Comment by CAnorbo** [comment]
@MarnixCroes thank you for the review, I've updated the files.

**Comment by MarnixCroes** [comment]
this has conflicts now

I bumped the client version in https://github.com/WalletWasabi/WasabiWalletWebSite/pull/41

File: wwwroot/index.html [hunk 4]
```diff
@@ -1,257 +1,480 @@
...
-                                    <br />This is why Wasabi Wallet is programmed to be a zero-knowledge software. Developers can't collect any sensitive information about you. What you do with your bitcoin is your business.
-                                </p>
-                            </div>
+                    </div>
+                    <div class="col-lg-4 mt-5 mt-lg-0">
+                        <div class="box">
+                            <h3 class="title-small position-relative ms-5 mt-4"> <img src="img/icon4.svg" class="icon-pull-left"/> SIMPLE </h3>
+                            <p class="ms-5"> Good privacy on bitcoin now just works. Use a simple yet functional wallet with all the features you'd expect. </p>
+                            <h3 class="title-small position-relative ms-5 mt-5"> <img src="img/icon5.svg" class="icon-pull-left"/> ACCESSIBLE </h3>
+                            <p class="ms-5"> Coinjoin with any amount, from 5,000 sats to 40,000 BTC and anything in between. WabiSabi is the coinjoin protocol under the hood that makes it work. </p>
```
Comment (MarnixCroes) [inline wwwroot/index.html:235]: this is all outdated, zkSNACKs biased

File: wwwroot/index.html [hunk 5]
```diff
@@ -1,257 +1,480 @@
...
-                                <p>Not your keys, not your coins. Wasabi Wallet lets you control your private keys, offering you true financial self sovereignty.</p>
-                            </li>
-                        </ul>
+            </div>
+        </section>
+        <section>
+            <div class="container">
+                <div class="row">
+                    <div class="col-lg-3 offset-lg-1">
+                        <h2 class="title">THE API FOR COINJOINS</h2>
```
Comment (MarnixCroes) [inline wwwroot/index.html:262]: this stuff about coinjoin API too

File: wwwroot/index.html [hunk 6]
```diff
@@ -1,257 +1,480 @@
...
+                            <label for="faq3">DO I NEED TO TRUST WASABI WITH MY COINS?</label>
+                            <div class="card-content">
+                                <p>No, Wasabi’s coinjoin implementation is trustless by design. The participants do not need to trust each other or the coordinator. Since only the user knows the private keys, only he can sign the transaction, which will only be done after verifying that everything is alright. Nobody can neither steal your coins, nor figure out which outputs belong to which inputs.</p>
+                            </div>
+                        </div>
+                        <div class="card">
+                            <input type="checkbox" id="faq4">
+                            <label for="faq4">DOES WASABI SUPPORT HARDWARE WALLETS?</label>
+                            <div class="card-content">
+                                <p>Wasabi does support hardware wallet usage through the standard Bitcoin-core HWI, although coinjoining straight from or to a hardware wallet is not yet possible.</p>
```
Comment (MarnixCroes) [inline wwwroot/index.html:359]: users can coinjoin to a hardware wallet now

File: wwwroot/index.html [hunk 7]
```diff
@@ -1,257 +1,480 @@
...
+                        <div class="floating-home">
+                            <div class="row">
+                                <div class="col-lg-5">
+                                    <h1 class="title"><strong>Wasabi Wallet</strong></h1>
+                                    <p class="mb-4">The most popular Bitcoin wallet for desktop with coinjoin. Free & open-source. Non-custodial.
+                                        <br>Reclaim your privacy today.</p><a href="index.html#download" class="btn btn-primary mt-2 mt-md-0">Download</a> <a href="#bitcoinisnotprivate" class="btn btn-blur affix-arrowright mt-2 mt-md-0">Read more</a> </div>
+                                <div class="col-lg-4 offset-lg-3">
+                                    <div class="box-blur d-none d-lg-block">
+                                        <div class="title-small mb-3">No default coordinator</div>
+                                        <p>Starting from June 1, 2024, Wasabi Wallet will no longer have a default coinjoin coordinator.</p><strong><a href="https://github.com/WalletWasabi" target="_blank" class="affix-arrowright">Don't trust, verify</a></strong> </div>
```
Comment (MarnixCroes) [inline wwwroot/index.html:130]: this should related to the announcement should link to the blog announcement not the github repo

**Review by MarnixCroes — CHANGES_REQUESTED** [review]
- PGP link should be updated too 
- API link in footer is dead
- what is the idea about the background image?

File: wwwroot/about.html [hunk 8]
```diff
@@ -226,86 +288,11 @@ <h3>Get started in 3 simple steps</h3>
...
-                                    <path class="cls-1" d="M99.41,0H39.59C17.73,0,0,17.73,0,39.59v59.81c0,21.87,17.73,39.59,39.59,39.59h59.81c21.87,0,39.59-17.73,39.59-39.59V39.59C139,17.73,121.27,0,99.41,0Zm13.42,39.96l-13.75,64.19s-.83,3.79-4,3.92c-.79,.03-1.57-.2-2.25-.62-4.83-2.95-22.58-16.21-22.58-16.21l-10.62,10.13s-1.93,1.93-3.98,2.05l1.69-21.96,39.57-35.47s.72-.97,0-1.21c-.72-.24-1.13-.56-3.34,.81-2.21,1.37-48.29,30.32-48.29,30.32l-21.49-6.51s-3.36-1.28-2.87-3.38c.36-1.56,2.41-2.65,3.86-3.13,1.45-.48,82.75-31.61,82.75-31.61,0,0,3.62-1.45,5.31,1.45,1.45,2.17,0,7.22,0,7.22Z" />
-                                </svg>
-                            </a>
-                        </li>
-                    </ul>
+                    <a href="#" target="_blank" class="white-text">Wasabi Wallet</a>
                 </div>
                 <div class="col-lg py-2 text-lg-end">
                     <a href="contribution.html" class="d-inline-block me-3">Contribution</a>
                     <a href="/swagger" class="d-inline-block me-3">API</a>
```
Comment (yahiheb) [inline wwwroot/about.html:295]: This should fix the broken API link in footer.

```suggestion
                    <a href="https://wasabiwallet.io/swagger/index.html" class="d-inline-block me-3">API</a>
```

File: wwwroot/buy-anything.html [hunk 9]
```diff
@@ -6,424 +6,410 @@
...
-                                    <path class="cls-1" d="M99.41,0H39.59C17.73,0,0,17.73,0,39.59v59.81c0,21.87,17.73,39.59,39.59,39.59h59.81c21.87,0,39.59-17.73,39.59-39.59V39.59C139,17.73,121.27,0,99.41,0Zm13.42,39.96l-13.75,64.19s-.83,3.79-4,3.92c-.79,.03-1.57-.2-2.25-.62-4.83-2.95-22.58-16.21-22.58-16.21l-10.62,10.13s-1.93,1.93-3.98,2.05l1.69-21.96,39.57-35.47s.72-.97,0-1.21c-.72-.24-1.13-.56-3.34,.81-2.21,1.37-48.29,30.32-48.29,30.32l-21.49-6.51s-3.36-1.28-2.87-3.38c.36-1.56,2.41-2.65,3.86-3.13,1.45-.48,82.75-31.61,82.75-31.61,0,0,3.62-1.45,5.31,1.45,1.45,2.17,0,7.22,0,7.22Z" />
-                                </svg>
-                            </a>
-                        </li>
-                    </ul>
+                    <a href="#" target="_blank" class="white-text">Wasabi Wallet</a>
                 </div>
                 <div class="col-lg py-2 text-lg-end">
                     <a href="contribution.html" class="d-inline-block me-3">Contribution</a>
                     <a href="/swagger" class="d-inline-block me-3">API</a>
```
Comment (yahiheb) [inline wwwroot/buy-anything.html:412]:
```suggestion
                    <a href="https://wasabiwallet.io/swagger/index.html" class="d-inline-block me-3">API</a>
```

File: wwwroot/contribution.html [hunk 10]
```diff
@@ -4,263 +4,248 @@
...
-                                    <path class="cls-1" d="M99.41,0H39.59C17.73,0,0,17.73,0,39.59v59.81c0,21.87,17.73,39.59,39.59,39.59h59.81c21.87,0,39.59-17.73,39.59-39.59V39.59C139,17.73,121.27,0,99.41,0Zm13.42,39.96l-13.75,64.19s-.83,3.79-4,3.92c-.79,.03-1.57-.2-2.25-.62-4.83-2.95-22.58-16.21-22.58-16.21l-10.62,10.13s-1.93,1.93-3.98,2.05l1.69-21.96,39.57-35.47s.72-.97,0-1.21c-.72-.24-1.13-.56-3.34,.81-2.21,1.37-48.29,30.32-48.29,30.32l-21.49-6.51s-3.36-1.28-2.87-3.38c.36-1.56,2.41-2.65,3.86-3.13,1.45-.48,82.75-31.61,82.75-31.61,0,0,3.62-1.45,5.31,1.45,1.45,2.17,0,7.22,0,7.22Z" />
-                                </svg>
-                            </a>
-                        </li>
-                    </ul>
+                    <a href="#" target="_blank" class="white-text">Wasabi Wallet</a>
                 </div>
                 <div class="col-lg py-2 text-lg-end">
                     <a href="contribution.html" class="d-inline-block me-3">Contribution</a>
                     <a href="/swagger" class="d-inline-block me-3">API</a>
```
Comment (yahiheb) [inline wwwroot/contribution.html:248]:
```suggestion
                    <a href="https://wasabiwallet.io/swagger/index.html" class="d-inline-block me-3">API</a>
```

File: wwwroot/faq.html [hunk 11]
```diff
@@ -442,86 +408,11 @@ <h3>Get started in 3 simple steps</h3>
...
-                                    <path class="cls-1" d="M99.41,0H39.59C17.73,0,0,17.73,0,39.59v59.81c0,21.87,17.73,39.59,39.59,39.59h59.81c21.87,0,39.59-17.73,39.59-39.59V39.59C139,17.73,121.27,0,99.41,0Zm13.42,39.96l-13.75,64.19s-.83,3.79-4,3.92c-.79,.03-1.57-.2-2.25-.62-4.83-2.95-22.58-16.21-22.58-16.21l-10.62,10.13s-1.93,1.93-3.98,2.05l1.69-21.96,39.57-35.47s.72-.97,0-1.21c-.72-.24-1.13-.56-3.34,.81-2.21,1.37-48.29,30.32-48.29,30.32l-21.49-6.51s-3.36-1.28-2.87-3.38c.36-1.56,2.41-2.65,3.86-3.13,1.45-.48,82.75-31.61,82.75-31.61,0,0,3.62-1.45,5.31,1.45,1.45,2.17,0,7.22,0,7.22Z" />
-                                </svg>
-                            </a>
-                        </li>
-                    </ul>
+                    <a href="#" target="_blank" class="white-text">Wasabi Wallet</a>
                 </div>
                 <div class="col-lg py-2 text-lg-end">
                     <a href="contribution.html" class="d-inline-block me-3">Contribution</a>
                     <a href="/swagger" class="d-inline-block me-3">API</a>
```
Comment (yahiheb) [inline wwwroot/faq.html:415]:
```suggestion
                    <a href="https://wasabiwallet.io/swagger/index.html" class="d-inline-block me-3">API</a>
```

File: wwwroot/index.html [hunk 12]
```diff
@@ -261,86 +459,10 @@ <h2 class="title title-lined">DOWNLOAD<br>WASABI WALLET</h2>
...
-                                    <path class="cls-1" d="M99.41,0H39.59C17.73,0,0,17.73,0,39.59v59.81c0,21.87,17.73,39.59,39.59,39.59h59.81c21.87,0,39.59-17.73,39.59-39.59V39.59C139,17.73,121.27,0,99.41,0Zm13.42,39.96l-13.75,64.19s-.83,3.79-4,3.92c-.79,.03-1.57-.2-2.25-.62-4.83-2.95-22.58-16.21-22.58-16.21l-10.62,10.13s-1.93,1.93-3.98,2.05l1.69-21.96,39.57-35.47s.72-.97,0-1.21c-.72-.24-1.13-.56-3.34,.81-2.21,1.37-48.29,30.32-48.29,30.32l-21.49-6.51s-3.36-1.28-2.87-3.38c.36-1.56,2.41-2.65,3.86-3.13,1.45-.48,82.75-31.61,82.75-31.61,0,0,3.62-1.45,5.31,1.45,1.45,2.17,0,7.22,0,7.22Z" />
-                                </svg>
-                            </a>
-                        </li>
-                    </ul>
+                    <a href="#" target="_blank" class="white-text">Wasabi Wallet</a>
                 </div>
                 <div class="col-lg py-2 text-lg-end">
                     <a href="contribution.html" class="d-inline-block me-3">Contribution</a>
-                    <a href="/swagger" class="d-inline-block me-3">API</a>
```
Comment (yahiheb) [inline wwwroot/index.html:340]: Why was the API removed here? It is present in all other pages.

File: wwwroot/index.html [hunk 13]
```diff
@@ -1,257 +1,455 @@
...
-                                    <br />This is why Wasabi Wallet is programmed to be a zero-knowledge software. Developers can't collect any sensitive information about you. What you do with your bitcoin is your business.
-                                </p>
-                            </div>
+                    </div>
+                    <div class="col-lg-4 mt-5 mt-lg-0">
+                        <div class="box">
+                            <h3 class="title-small position-relative ms-5 mt-4"> <img src="img/icon4.svg" class="icon-pull-left"/> SIMPLE </h3>
+                            <p class="ms-5">With Wasabi Wallet good privacy on bitcoin just works. Use a simple yet functional wallet with all the features you'd expect. </p>
+                            <h3 class="title-small position-relative ms-5 mt-5"> <img src="img/icon5.svg" class="icon-pull-left"/> ACCESSIBLE </h3>
+                            <p class="ms-5">Coinjoin with any amount, from 5,000 sats to 40,000 BTC and anything in between. WabiSabi is the coinjoin protocol under the hood that makes it work. </p>
```
Comment (yahiheb) [inline wwwroot/index.html:235]: As Marnix pointed out, this is all outdated, zkSNACKs biased.

File: wwwroot/faq.html [hunk 14]
```diff
@@ -1,427 +1,393 @@
...
+            "@type": "Question",
+            "name": " What is a “coinjoin”?",
+            "acceptedAnswer": {
+              "@type": "Answer",
+              "text": "<p>Coinjoin is a collaborative transaction between multiple peers.</p><p>Usually, but not necessarily, it consists of some standard output denominations that users should break their coins in to. This makes it difficult for outside parties to trace where a particular coin was sent to (as opposed to regular bitcoin transactions, where there is usually one sender and one receiver).</p><p>Coinjoins can be done with non-custodial software like Wasabi Wallet, that eliminates the risk of funds disappearing or being stolen. The funds will always be in a bitcoin address that the user controls and not even developers can alter the transaction or redirect the funds.</p><p>Coinjoin basicly means: “when you want to make a transaction, find someone else who also wants to make a transaction and make a joint transaction together”.</p>"
+            }
+          },
+          {
+            "@type": "Question",
+            "name": "What are the fees for coinjoins?",
```
Comment (yahiheb) [inline wwwroot/faq.html:46]: This should be removed.

**Review by yahiheb — CHANGES_REQUESTED** [review]
- Why was the onion-index file removed?
- What is the idea about the background image?
- I don't think we need a background video for the 404 page.
- The video `homepage_anim2.mp4` is not used anywhere. Such huge file should be just removed.
- Idk if all new added files are used or not, I didn't check that.
- If you click MENU on other pages like FAQ you can notice this:
image

File: wwwroot/faq.html [hunk 15]
```diff
@@ -1,427 +1,393 @@
...
+            "acceptedAnswer": {
+              "@type": "Answer",
+              "text": "<p>Coinjoining coins with a value above 0.01 BTC costs 0.3% as a coordinator fee + mining fees. Inputs of 0.01 BTC or below don't pay coordinator fees, nor remixes, even after one transaction. Thus, a payment made with coinjoined funds allows the sender and the recipient to remix their coins without paying any coordinator fees.</p>"
+            }
+          },
+          {
+            "@type": "Question",
+            "name": "Do I need to trust the coordinator with my coins?",
```
[the remark this answers is not in the document]
Comment (CAnorbo) [reply wwwroot/faq.html:54]: Please send me the other Question + Answer.
Btw Max asked this change.

File: wwwroot/index.html (hunk 12 above)
Comment (CAnorbo) [reply wwwroot/index.html:340]: Max asked it, but I've put back and fixed to the right one.

**Comment by CAnorbo** [comment]
> * Why was the onion-index file removed?
> * What is the idea about the background image?
> * I don't think we need a background video for the 404 page.
> * The video `homepage_anim2.mp4` is not used anywhere. Such huge file should be just removed.
> * Idk if all new added files are used or not, I didn't check that.
> * If you click MENU on other pages like FAQ you can notice this:
>   image

- The onion file is pointless because it's not used anywhere for long time. Tor also using the index.html. 
- The background image is temporary until we'll have the final video here, but it's not ready yet.
- Removed the video files and also the background video from the 404 page.
- I couldn't reproduce the menu issue, in which browser you did the testing?

**Review by yahiheb — CHANGES_REQUESTED** [review]
> I couldn't reproduce the menu issue, in which browser you did the testing?

Chrome.

- These files are not used: `icon_white1.svg`, `icon_white2.svg`, `icon_white3.svg`, `logo_small_alt.svg`
- Just curious, why all the FAQs in the FAQ page are repeated twice?

**Comment by CAnorbo** [comment]
> > I couldn't reproduce the menu issue, in which browser you did the testing?
> 
> Chrome.
> 
> * These files are not used: `icon_white1.svg`, `icon_white2.svg`, `icon_white3.svg`, `logo_small_alt.svg`
> * Just curious, why all the FAQs in the FAQ page are repeated twice?

One is meta data for search engines and the other is the real content.

File: wwwroot/faq.html [hunk 16]
```diff
@@ -1,427 +1,400 @@
...
+            "@type": "Question",
+            "name": "What is the passphrase used for?",
+            "acceptedAnswer": {
+              "@type": "Answer",
+              "text": "<p>The passhprase you set is used as a 13th seed word (as described in <a href='https://github.com/bitcoin/bips/blob/master/bip-0039.mediawiki' target='_blank'>BIP 39</a>) and to encrypt the private key of the extended private key (as described in <a href='https://github.com/bitcoin/bips/blob/master/bip-0038.mediawiki' target='_blank'>BIP 38</a>) to get an encrypted secret which is stored on the computer.</p><p>Wasabi Wallet stores only the BIP38 encrypted blob, so you'll need to type in the passphrase to spend or coinjoin from the wallet.</p><p>The passphrase will unlock your bitcoin to anyone who has access to the recovery words backup or the computer! If your backup gets compromised, this passphrase is the only thing protecting your precious sats.</p><p>It is important to use a random and long passphrase.</p>"
+            }
+          },
+          {
+            "@type": "Question",
+            "name": "What are the fees for coinjoins?",
```
Comment (yahiheb) [inline wwwroot/faq.html:110]: IMO this should be moved up to be the 2nd FAQ.
If it doesn't matter just ignore my comment.

File: wwwroot/faq.html [hunk 17]
```diff
@@ -1,427 +1,400 @@
...
+            "acceptedAnswer": {
+              "@type": "Answer",
+              "text": "The coinjoin coordinator sets its own fee policy for the coinjoin service provided.This includes things like the mining fee, the coordination fee, no coordination fee for remixes and/or no coordination fee for UTXO amounts below a threshold etc."
```
Comment (yahiheb) [inline wwwroot/faq.html:113]:
```suggestion
              "text": "The coinjoin coordinator sets its own fee policy for the coinjoin service provided. This includes things like the mining fee, the coordination fee, no coordination fee for remixes and/or no coordination fee for UTXO amounts below a threshold etc."
```

**Review by yahiheb — CHANGES_REQUESTED** [review]
`screen_shot1.png` screenshot should be updated because we no longer have `Add Password` dialog, instead we have `Add passphrase`.

Did you manage to reproduce [this](https://github.com/WalletWasabi/WasabiWalletWebSite/pull/40#pullrequestreview-2102928225):
> If you click MENU on other pages like FAQ you can notice this:

Other than these suggestions, LGTM.

File: wwwroot/index.html [hunk 18]
```diff
@@ -1,198 +1,385 @@
...
-                                    We live in an Orwellian surveillance society where your information is being used to typecast and manipulate you. Bitcoin projects are being pressured to collect more and more data, if possible.
-                                    <br />This is why Wasabi Wallet is programmed to be a zero-knowledge software. Developers can't collect any sensitive information about you. What you do with your bitcoin is your business.
-                                </p>
+                <div class="row vh-100">
+                    <div class="col-lg-11 offset-lg-1 position-relative">
+                        <div class="floating-home">
+                            <div class="row">
+                                <div class="col-lg-5">
+                                    <h1 class="title"><strong>Wasabi Wallet</strong></h1>
+                                    <p class="mb-4">The most popular Bitcoin wallet for desktop with coinjoin. Free & open-source. Non-custodial.
```
Comment (MarnixCroes) [inline wwwroot/index.html:125]: "the most popular" is not professional, should be rephrased

File: wwwroot/index.html [hunk 19]
```diff
@@ -1,198 +1,385 @@
...
+                                        <br>Reclaim your privacy today.</p><a href="index.html#download" class="btn btn-primary mt-2 mt-md-0">Download</a> <a href="#bitcoinisnotprivate" class="btn btn-blur affix-arrowright mt-2 mt-md-0">Read more</a> </div>
+                                <div class="col-lg-4 offset-lg-3">
+                                    <div class="box-blur d-none d-lg-block">
+                                        <div class="title-small mb-3">No default coordinator</div>
+                                        <p>Starting from June 1, 2024, Wasabi Wallet will no longer have a default coinjoin coordinator.</p><strong><a href="https://blog.wasabiwallet.io/zksnacks-is-discontinuing-its-coinjoin-coordination-service-1st-of-june/" target="_blank" class="affix-arrowright">Don't trust, verify</a></strong> </div>
```
Comment (MarnixCroes) [inline wwwroot/index.html:130]: the link to the shutdown announcement being "don't trust, verify" is weird
better rephrase it

File: wwwroot/index.html [hunk 20]
```diff
@@ -1,198 +1,385 @@
...
+            </div>
+        </section>
+        <section>
+            <div class="container">
+                <div class="row">
+                    <div class="col-lg-3 offset-lg-1">
+                        <h2 class="title">Privacy by default</h2>
+                    </div>
+                    <div class="col-lg-4">
+                        <p> Get started with good privacy by default. All traffic is routed via Tor. Your wallet transactions are downloaded with block filters. Easily reclaim your privacy with collaborative bitcoin transactions. </p>
```
Comment (MarnixCroes) [inline wwwroot/index.html:206]:
```suggestion
                        <p> Get started with privacy by default. All traffic is routed via Tor by default. Your wallet is synchronized using client-side block filters. Gain on-chain privacy with the built-in coinjoins. </p>
```
this is a bit weird structure imo. my proposal is more of a fix, maybe rewrite is better

File: wwwroot/index.html [hunk 21]
```diff
@@ -1,198 +1,385 @@
...
+        <section>
+            <div class="container">
+                <div class="row align-items-end">
+                    <div class="col-lg-8 offset-lg-1">
+                        <div class="row">
+                            <div class="col-lg-6">
+                                <h2 class="title">Mining fees and coordinator fees</h2>
+                            </div>
+                            <div class="col-lg-6">
+                                <p>Every individual user is responsible for paying their own mining fee for the block space they utilize. This arrangement guarantees that the expenses associated with transactions are equitably allocated among all users, ensuring fairness in the distribution of costs.</p>
```
Comment (MarnixCroes) [inline wwwroot/index.html:267]: lettings users pay for their blockspace, a.k.a. everyone responsible for their own mining fee is also a coordinator policy

File: wwwroot/index.html [hunk 22]
```diff
@@ -1,198 +1,385 @@
...
+                            <label for="faq3">DO I NEED TO TRUST THE COORDINATOR WITH MY COINS?</label>
+                            <div class="card-content">
+                                <p>No, Wasabi’s coinjoin implementation is trustless by design. The participants do not need to trust each other or the coordinator. Since only the user knows the private keys, only he can sign the transaction, which will only be done after verifying that everything is alright. Nobody can neither steal your coins, nor figure out which outputs belong to which inputs.</p>
+                            </div>
+                        </div>
+                        <div class="card">
+                            <input type="checkbox" id="faq4">
+                            <label for="faq4">DOES WASABI SUPPORT HARDWARE WALLETS?</label>
+                            <div class="card-content">
+                                <p>Wasabi does support hardware wallet usage through the standard Bitcoin-core HWI, and coinjoining straight to a hardware wallet is possible, but signing a coinjoin trasaction with a hardwarewallet is not implemented.</p>
```
Comment (MarnixCroes) [inline wwwroot/index.html:303]:
```suggestion
                                <p>Wasabi does support hardware wallet usage through the standard Bitcoin-core HWI, and coinjoining straight to a hardware wallet is possible, but signing a coinjoin trasaction with a hardware wallet is not implemented.</p>
```

File: wwwroot/index.html [hunk 23]
```diff
@@ -1,198 +1,385 @@
...
+                        <div class="row align-items-top mb-5">
+                            <div class="col-lg-6 order-1">
+                                <h3 class="title-huge">DOWNLOAD</h3>
+                            </div>
+                            <div class="col-lg-6 order-3 order-lg-2">
+                                <div class="ps-0 ps-lg-5">
+                                    <p class="small-text"><a href="https://github.com/WalletWasabi/WalletWasabi#build-from-source-code" target="_blank"><i class="muted-text">or build it from source</i></a>
+                                        <br><a href="https://raw.githubusercontent.com/WalletWasabi/WalletWasabi/master/PGP.txt" target="_blank">PGP: 6FB3 872B 5D42 292F 5992 0797 8563 4832 8949 861E</a>
+                                    </p>
+                                    <a href="https://github.com/WalletWasabi/releases" target="_blank" class="affix-arrowright">RELEASE NOTES</a>
```
Comment (MarnixCroes) [inline wwwroot/index.html:372]:
```suggestion
                                    <a href="https://github.com/WalletWasabi/WalletWasabi/releases" target="_blank" class="affix-arrowright">RELEASE NOTES</a>
```

**Review by MarnixCroes — COMMENTED** [review]
for the `screen_shot1.png`, _Add password_ is old, it is _Add Passphrase_ now

**Review by yahiheb — CHANGES_REQUESTED** [review]

**Comment by turbolay** [comment]
Thank you, everything was fixed except:
- Menu doesn't close automatically when click outside of it or click on a link that doesn't change page


I made an ultimate round of review, I found few other things (sorry)
- Menu is too big on mobile, it takes the whole page
- Menu background is not opaque enough on mobile, please use same opacity than desktop
See both problems on this screenshot:

CleanShot 2024-06-24 at 13 45 11@2x

- within 2 certain screen width, the `Other Linux` installation box takes all the screen which is ugly

CleanShot 2024-06-24 at 13 45 56@2x

- In the left sidebar, swap Reddit with Youtube please, it would group all community channels to the end which is good

-------

After those items are done, please merge

**Comment by CAnorbo** [comment]
Close menu is not possible without JS.

**Comment by turbolay** [comment]
Ok, thanks for the fixes.

**Review by turbolay — APPROVED** [review]

**Comment by CAnorbo** [comment]
I removed the link from the Title. That shouldn't be a Back to top button.

**Pull request merged**